### ISSUE TITLE
fixed color show

### ISSUE THREAD

**Issue by newyorkthink** [jesseduffield/lazydocker#485]
![image](https://github.com/jesseduffield/lazydocker/assets/87791643/299f51b9-f8cc-44e2-905e-16a5ad391419)

when i select images , the background color is full line

my terminal theme is nice, but lazydocker is Cannnot fix this

**Comment by jorgemxm** [comment]
Hi @newyorkthink I was curious about this bug you mentioned,
Are you referring to the Background color of the selected line? is it about the text being unreadable?

(If yes that both text and background colors can be customized from lazydocker's config file)
Cheers

**Comment by newyorkthink** [comment]
> unreadable

yes .cannot unreadable , and i customized background , is not work

**Comment by jorgemxm** [comment]
Hi @newyorkthink I believe you could edit lazydocker's config.yml file and set a different background color for the selected line using `selectedLineBgColor:`  You could use a web color such as `#261268` or one of the named colors, for instance, try "black" as the value

Here you can see more info about the config options: https://github.com/jesseduffield/lazydocker/blob/master/docs/Config.md

Config example:
<img width="851" alt="image" src="https://github.com/jesseduffield/lazydocker/assets/5948446/13662b49-7aab-4da9-9602-d2e882407731">

**Comment by newyorkthink** [comment]
cool, the bug is fixed
it can see
![image](https://github.com/jesseduffield/lazydocker/assets/87791643/5928eb96-3d23-4c7e-8464-91555ccb0132)
